src/rust/sdk/src/lib.rs

```diff
@@ -187,13 +196,23 @@ impl EconiaClient {
             .map(|r| r.is_some())
     }
 
+    /// Checks if the user is registered for a coin.
+    ///
+    /// # Arguments:
+    ///
+    /// * `coin` - Aptos [TypeTag] for the given coin.
```

Comment:
Can we disambiguate here versus a `MarketAccount`?

Checks if the user is registered with a [CoinStore]